src/Symfony/Component/Validator/ValidatorBuilder.php

```diff
@@ -256,8 +256,11 @@ public function setConstraintValidatorFactory(ConstraintValidatorFactoryInterfac
     /**
      * {@inheritdoc}
      */
-    public function setTranslator(LegacyTranslatorInterface $translator)
+    public function setTranslator($translator)
```

Comment:
Changing this is a BC break - but that might be OK as maybe nobody extends this class.
If not, we need to rethink the BC layer.